fabric-network/types/index.d.ts

```diff
@@ -45,6 +45,11 @@ export interface DiscoveryOptions {
 	enabled?: boolean;
 }
 
+export interface DiscoveryInterest {
+	name: string;
+	collection_names?: string[];
```

Comment:
snake_case is inconsistent with camelCase used everywhere else in the _fabric-network_ API. Even if _fabric-common_ is using snake_case, we could translate for the user so _fabric-network_ consistently uses camelCase.

Reply:
fixed